runtime/src/commonMain/kotlin/pbandk/Export.kt

```diff
@@ -0,0 +1,14 @@
+package pbandk
+
+@Target(AnnotationTarget.CLASS, AnnotationTarget.PROPERTY, AnnotationTarget.FUNCTION, AnnotationTarget.FILE)
+expect annotation class Export()
+
+@Target(
+    AnnotationTarget.CLASS,
+    AnnotationTarget.FUNCTION,
+    AnnotationTarget.PROPERTY,
+    AnnotationTarget.CONSTRUCTOR,
+    AnnotationTarget.PROPERTY_GETTER,
+    AnnotationTarget.PROPERTY_SETTER
+)
+expect annotation class Name(val name: String)
```

Comment:
Whereas it makes sense for `pbandk.Export` to be language-neutral (rather than naming it `pbandk.JsExport`), I'd recommend naming this annotation `pbandk.JsName`.

There's also a `JvmName` annotation in Kotlin and the reasons for having to rename a function are different in JVM and JS. So there's a good chance we'd want to have a `pbandk.JvmName` annotation in the future and use different names for Kotlin code, JS code, and JVM code.